packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -575,17 +660,39 @@ class _RenderableImageLayer extends _RenderableLayer<ImageLayer> {
 
   static Future<_RenderableLayer> load(
     ImageLayer layer,
+    _RenderableGroupLayer? parent,
     Camera? camera,
   ) async {
     return _RenderableImageLayer(
       layer,
+      parent,
       await Flame.images.load(layer.image.source!),
     );
   }
 }
 
+class _RenderableGroupLayer extends _RenderableLayer<Group> {
+  /// The child layers of this [Group] to be rendered recursively.
+  ///
+  /// NOTE: This is set externally instead of via constructor params because
+  ///       there are cyclic dependencies when loading the renderable layers.
+  late final List<_RenderableLayer> children;
+
+  _RenderableGroupLayer(
+    super.layer,
+    super.parent,
+  );
+
+  @override
+  void render(ui.Canvas canvas, Camera? camera) {
+    children.forEach((child) {
```

Comment:
dart style: [avoid forEach with function literal](https://dart.dev/guides/language/effective-dart/usage#avoid-using-iterableforeach-with-a-function-literal)

for (final child in children) child.render(canvas, camera);

Reply:
Pushing a fix, thanks for the link to the style guide